handlers/accesscontrol/access-control.go

```diff
@@ -198,26 +251,44 @@ func (ac *AccessControlHandlersCollection) cachePlaybackAccessControlInfo(playba
 	return nil
 }
 
-func (g *GateClient) QueryGate(body []byte) (bool, int32, int32, error) {
+func (g *GateClient) QueryGate(body []byte) (bool, int32, int32, int32, error) {
```

Comment:
Could you add a comment on what these return params are please?

Reply:
Sure, adding them

Considering that they grew and soon they are going to include other params (refreshInterval, for example) I want to put them into a separate `type` in the next PR